service/src/integrationTest/kotlin/fi/espoo/evaka/occupancy/PlannedOccupancyTest.kt

```diff
@@ -37,7 +48,9 @@ class PlannedOccupancyTest : FullApplicationTest() {
         jdbi.handle(::resetDatabase)
     }
 
-    private val defaultPeriod = ClosedPeriod(LocalDate.of(2019, 1, 1), LocalDate.of(2019, 12, 31))
+    private val defaultPeriod = ClosedPeriod(LocalDate.of(2019, 1, 1), LocalDate.of(2019, 1, 31))
+    val period1 = ClosedPeriod(defaultPeriod.start, LocalDate.of(2019, 1, 15))
+    val period2 = ClosedPeriod(LocalDate.of(2019, 1, 16), defaultPeriod.end)
```

Comment:
This is very minor, but should these be named something like `defaultPeriodSplit1` and `defaultPeriodSplit2` so it's immediately apparent that they are two halves of `defaultPeriod`?